plugin-server/src/worker/ingestion/team-manager.ts

```diff
@@ -215,13 +215,16 @@ DO UPDATE SET property_type=$5 WHERE posthog_propertydefinition.property_type IS
             )
             const distinctIds: { distinct_id: string }[] = organizationMembers.rows
             for (const { distinct_id } of distinctIds) {
-                posthog.identify(distinct_id)
-                posthog.capture('first team event ingested', {
-                    team: team.uuid,
-                    sdk: properties.$lib,
-                    realm: properties.realm,
-                    host: properties.$host,
-                    $groups: {
+                posthog.capture({
```

Comment:
no need to identify?

Reply:
I don't think so no. We aren't saving any properties against a user and the distinctId is the distinctId :D 

Identify never made sense for the way it was implemented before as it was caching the distinctId for all subsequent calls :o and doing aliasing and all sorts of bad stuff. As we are capturing a fresh event

Reply:
The old js-lite was working as a client lib here when it should have been behaving like a server-side lib